rir/src/compiler/translations/rir_2_pir/rir_2_pir.cpp

```diff
@@ -272,12 +272,14 @@ bool Rir2Pir::compileBC(const BC& bc, Opcode* pos, rir::Code* srcCode,
 
         auto ast = bc.immediate.callFixedArgs.ast;
         auto insertGenericCall = [&]() {
-            if (bc.bc == Opcode::named_call_implicit_)
+            if (bc.bc == Opcode::named_call_implicit_) {
```

Comment:
Why do the named calls not have a frame state?

Reply:
because we do not inline them